hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -117,6 +117,10 @@
   public static final String MAX_CONSISTENCY_CHECKS_PROP = "hoodie.consistency.check.max_checks";
   public static int DEFAULT_MAX_CONSISTENCY_CHECKS = 7;
 
+  // Data loss check before commits
+  private static final String DATALOSS_CHECK_ENABLED = "hoodie.dataloss.check.enabled";
```

Comment:
let's name this specific to real purpose like. `hoodie.merge.data.validation.enabled` , avoiding the calling this loss checking etc, which can be rather disconcerting to users, when they read this.

Reply:
+1

Reply:
Good suggestion.

Reply:
+1